File: 1929.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
const getConcatenation = function(nums) {
    const res = [];
    const numsLength = nums.length;
    for (let i = 0; i < numsLength*2; i++) {
        if (i >= numsLength) {
            res.push(nums[i % numsLength])
            continue;
        }
        res.push(nums[i]);
    }
    return res;
};

File: 1929.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        res = []
        numsLength = len(nums)
        for i in range(numsLength*2):
            if (i >= numsLength):
                res.append(nums[i % numsLength])
                continue
            res.append(nums[i])
        return res